### PR TITLE
Adapt w.r.t. coq/coq#16004.

### PR DESCRIPTION
**Pull request by ppedrot**
I've done this one by hand because it's really small. @JasonGross I am unsure of the amount of backwards compatibility you want to preserve, but the only two problematic attributes are the ones for the `Hint Rewrite` commands. Not sure if you want to add a script to simply remove them on old versions, or do something more clever.

### CODE REVIEW

**Comment by JasonGross** [comment]
The CI only tests master and 8.11, so let's just add unsupported-attributes to
https://github.com/mit-plv/engine-bench/blob/d2cf8602d479b627c1f5d3d03044e8570af827ea/coq/PerformanceExperiments/Makefile#L28
and
https://github.com/mit-plv/engine-bench/blob/d2cf8602d479b627c1f5d3d03044e8570af827ea/coq/PerformanceDemos/Makefile#L19

**Comment by ppedrot** [comment]
@JasonGross Done.

**Comment by ppedrot** [comment]
@JasonGross ping. This is the last dev whose overlay is missing for coq/coq#16004.

**Comment by ppedrot** [comment]
@JasonGross pingy-ping.

**Comment by JasonGross** [comment]
Sorry, currently busy traveling.  I hope to get to this by the end of the week.  I think I need to switch the dev action to a docker container?  I'm not fully sure what's going wrong

**Comment by JasonGross** [comment]
Ah, no, I think I figured it out.  Try rebasing on master now

File: coq/PerformanceDemos/Makefile [hunk 1]
```diff
@@ -16,7 +16,7 @@ all:: invoke-coqmakefile
 
 include ../../Makefile.show
 
-WARNINGS:=-notation-overridden
+WARNINGS:=-notation-overridden,-unsupported-attributes
```
Comment (JasonGross) [inline coq/PerformanceDemos/Makefile:19]: You need to `make update-_CoqProject` to get this change propagated to the checked-in _CoqProject

**Review by JasonGross — CHANGES_REQUESTED** [review]

**Pull request merged**